src/ore/src/ssh_key.rs

```diff
@@ -15,17 +15,20 @@
 
 //! Utilities for generating and managing SSH keys.
 
+use std::fmt;
+
 use openssl::pkey::{PKey, Private};
+use serde::ser::{SerializeStruct, Serializer};
 use serde::{Deserialize, Serialize};
 use ssh_key::private::{Ed25519Keypair, Ed25519PrivateKey, KeypairData};
 use ssh_key::public::Ed25519PublicKey;
 use ssh_key::{LineEnding, PrivateKey};
 
 /// Keeps track of a public-private keypair, stored in the OpenSSH format
-#[derive(Debug, Eq, PartialEq, PartialOrd, Ord, Clone, Serialize, Deserialize)]
+#[derive(Debug, Clone)]
 pub struct SshKeypair {
-    public_key: Vec<u8>,
-    private_key: Vec<u8>,
+    // Even though the type is called PrivateKey, it includes the full keypair
+    keypair: PrivateKey,
```

Comment:
```suggestion
    // Even though the type is called PrivateKey, it includes the full keypair,
    // and zero's memory on `Drop`
    keypair: PrivateKey,
```
last I read the impl this was true, right?

Reply:
Yep!